src/createServer.js

```diff
@@ -1,7 +1,17 @@
 'use strict';
 
+const express = require('express');
+
+const { router: userRouter } = require('./routers/user.route');
+const { router: expenseRouter } = require('./routers/expense.route');
+
 const createServer = () => {
-  // your code goes here
+  const app = express();
+
+  app.use('/users', express.json(), userRouter);
+  app.use('/expenses', express.json(), expenseRouter);
```

Comment:
```suggestion
  app.use(express.json());
  
  app.use('/users', userRouter);
  app.use('/expenses', expenseRouter);
```
You can apply middleware once for all routes